debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/MySqlConnectorConfig.java

```diff
@@ -817,6 +836,17 @@ private static int validateGtidSetExcludes(Configuration config, Field field, Va
         return 0;
     }
 
+    private static int validateGtidSourceFilterDmlEvents(Configuration config, Field field, ValidationOutput problems) {
+        String includes = config.getString(GTID_SOURCE_INCLUDES);
+        String excludes = config.getString(GTID_SOURCE_EXCLUDES);
+        Boolean filterDmlEvents = config.getBoolean(GTID_SOURCE_FILTER_DML_EVENTS);
+        if (includes == null && excludes == null) {
+            problems.accept(GTID_SOURCE_FILTER_DML_EVENTS, filterDmlEvents , "Either gtid.source.includes or gtid.source.excludes must be specified.");
+            return 1;
```

Comment:
Should this problem be reported only if the boolean value of `filterDmlEvents` is true?